src/datastore/postgres-store.ts

```diff
@@ -1915,9 +1918,13 @@ export class PgDataStore extends (EventEmitter as { new (): DataStoreEventEmitte
       text: insertQuery,
       values,
     };
-    const res = await client.query(insertBNSSubdomainsEventQuery);
-    if (res.rowCount !== subdomains.length) {
-      throw new Error(`Expected ${subdomains.length} inserts, got ${res.rowCount}`);
+    try {
+      const res = await client.query(insertBNSSubdomainsEventQuery);
+      if (res.rowCount !== subdomains.length) {
+        throw new Error(`Expected ${subdomains.length} inserts, got ${res.rowCount}`);
+      }
+    } catch (e) {
+      console.log('subdomain errors', e.message);
```

Comment:
If there's no way to recover from this error it should be fatal (remove the try/catch)